onnx/src/jvmMain/kotlin/org/jetbrains/kotlinx/dl/onnx/inference/facealignment/Fan2D106FaceAlignmentModel.kt

```diff
@@ -21,7 +21,7 @@ import java.io.IOException
 private const val OUTPUT_NAME = "fc1"
 
 /**
- * The light-weight API for solving Face Alignment task via Fan2D106 model.
+ * The light-weight API for solving a Face Alignment task via Fan2D106 model.
```

Comment:
To me indefinite article seems inappropriate with the capital letters. Perhaps, "face alignment tasks" (multiple)?